docs/dg/dev/backend-development/data-manipulation/data-publishing/implement-publish-and-synchronization.md

```diff
@@ -43,7 +43,7 @@ console code:generate:module:shared MyModuleStorage
 
 {% info_block infoBox "Naming convention"%}
 
-As a naming convention, names of modules that publish data to Redis end with Storage (for example, `MyModuleStorage`, and names of modules that publish to Elasticsearch end with Search (for example, `MyModuleSearch`).
```

Comment:
we might have conflicts here https://github.com/spryker/spryker-docs/pull/3185

Reply:
I would update naming in my document, so we can release it together or we can just rollback yours and release mine after

Reply:
Reverted, please, don`t forget to rename in your doc